src/views/Transitioner.js

```diff
@@ -48,6 +48,16 @@ type State = {
   scenes: Array<NavigationScene>,
 };
 
+type Scenes = Array<Scene>;
+
+type Scene = {
```

Comment:
You should use the existing type from `TypeDefinition` instead of re-declaring it here